src/tests/unit/tests/DetailsView/components/preview-features-toggle-list.test.tsx

```diff
@@ -32,37 +33,13 @@ describe('PreviewFeaturesToggleListTest', () => {
                 enabled: false,
             },
         ];
-        const setFeatureFlagStub = () => {};
         const actionMessageCreatorMock = Mock.ofType(DetailsViewActionMessageCreator);
         const props: PreviewFeaturesToggleListProps = {
             displayedFeatureFlags: displayableFeatureFlagsStub,
             actionMessageCreator: actionMessageCreatorMock.object,
         };
-        const testSubject = new PreviewFeaturesToggleList(props);
 
-        actionMessageCreatorMock.setup(acm => acm.setFeatureFlag).returns(() => setFeatureFlagStub);
-
-        const expectedComponent = (
-            <div className="preview-feature-toggle-list">
-                <GenericToggle
-                    name={props.displayedFeatureFlags[0].displayableName}
-                    description={props.displayedFeatureFlags[0].displayableDescription}
-                    enabled={props.displayedFeatureFlags[0].enabled}
-                    onClick={setFeatureFlagStub}
-                    id={props.displayedFeatureFlags[0].id}
-                    key={`preview_feature_toggle${props.displayedFeatureFlags[0].id}`}
-                />
-                <GenericToggle
-                    name={props.displayedFeatureFlags[1].displayableName}
-                    description={props.displayedFeatureFlags[1].displayableDescription}
-                    enabled={props.displayedFeatureFlags[1].enabled}
-                    onClick={setFeatureFlagStub}
-                    id={props.displayedFeatureFlags[1].id}
-                    key={`preview_feature_toggle${props.displayedFeatureFlags[1].id}`}
-                />
-            </div>
-        );
-
-        expect(testSubject.render()).toEqual(expectedComponent);
+        const testSubject = shallow(<PreviewFeaturesToggleList {...props} />);
```

Comment:
> c [](start = 7, length = 2)

with this change, we don't validate if onclick is setup properly rite?

before this change:
actionMessageCreator.setFeatureFlag is tested (in another ut file)
onClick=actionMessageCreator.setFeatureFlag